lib/presentation/atoms/timer_atom.dart

```diff
@@ -70,7 +70,9 @@ class TimerAtom extends StatelessWidget {
                                   )
                                 : null,
                             child: TextAtom(
-                              "${time.minutes}:${time.seconds}",
+                              time.hours == "00"
+                                ? "${time.minutes}:${time.seconds}"
+                                : "${time.hours}:${time.minutes}:${time.seconds}",
```

Comment:
I prefer not to have duplicated code

`"${time.hours == '00' ? '' : '${time.hours}:'}${time.minutes}:${time.seconds}",`